terminado/websocket.py

```diff
@@ -56,6 +63,26 @@ def initialize(self, term_manager):
 
         self._logger = logging.getLogger(__name__)
 
+        self.keep_alive_time = 5 #send a ping if there is no activity for more than this many of seconds
+        self.last_activity_time = keep_alive_time 
```

Comment:
These assignments don't look quite right - shouldn't it be `self.keep_alive_time = keep_alive_time`? And I guess `last_activity_time` should be set to 0 here.